src/common/common.js

```diff
@@ -732,7 +732,7 @@ const MessageHandler = (function () {// eslint-disable-line no-unused-vars
         if (typeof args[0] === "boolean") {
             isDismissable = args.shift();
         }
-        if (typeof args[0] !== undefined && args[0].text !== undefined && args[0].action !== undefined) {
+        if (typeof args[0] !== "undefined" && args[0].text !== "undefined" && args[0].action !== "undefined") {
```

Comment:
In ES6 it is no longer needed to compare the typeof value to the string of `undefined` explicitly. See https://stackoverflow.com/questions/34596489/es2015-2016-way-of-typeof-varname-undefined#34596516

Reply:
Mmh... But it gave me an error and when I printed `typeof args[0] !== undefined` it returned `false` even if the array was empty. And `typeof args[0] !== "undefined"` returned `true`...

Reply:
Firefox 60:

![grafik](https://user-images.githubusercontent.com/11966684/40268746-7b9e85de-5b73-11e8-9282-b294465bb710.png)

Reply:
And and yeah… it's wrong. As https://stackoverflow.com/questions/34596489/es2015-2016-way-of-typeof-varname-undefined#34596516 says, we may better not use that "typeof" for it anyway. In ES6 one can just check whether it is undefined directly.
So better not use `typeof`.

Reply:
Alright. I will change that.